pipx/action.yml

```diff
@@ -44,14 +44,32 @@ runs:
       id: python
       with:
         python-version: ${{ inputs.python-version }}
+    - name: Pipx version
+      id: version
+      run: |
+        PIPX_VERSION=$(pipx --version)
+
+        IFS='.' read -r major minor patch << EOF
+        $PIPX_VERSION
+        EOF
+
+        echo "version=$PIPX_VERSION" >> $GITHUB_OUTPUT
+        echo "major=$major" >> $GITHUB_OUTPUT
+        echo "minor=$minor" >> $GITHUB_OUTPUT
+        echo "patch=$patch" >> $GITHUB_OUTPUT
+      shell: bash
     - name: Get pipx settings
       id: settings
       run: |
-        pipx environment
-        echo "home=$(pipx environment -v PIPX_HOME)" >> $GITHUB_OUTPUT
-        echo "bin=$(pipx environment -v PIPX_BIN_DIR)" >> $GITHUB_OUTPUT
-        echo "venvs=$(pipx environment -v PIPX_LOCAL_VENVS)" >> $GITHUB_OUTPUT
-        echo "shared=$(pipx environment -v PIPX_SHARED_LIBS)" >> $GITHUB_OUTPUT
+        if [ "${{ steps.version.outputs.major }}" -ge 2 -a "${{ steps.version.outputs.major }}" -ge 1 ]; then
```

Comment:
This?     
   if [ "${{ steps.version.outputs.major }}" -ge 2 -a "${{ steps.version.outputs.minor }}" -ge 1 ]; then